recipes/hidefix/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "hidefix" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: 9ac6c81783e1d8c401c4c5bd2d2df1f6f85bc91de32889d54acf62506bbd6910
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make
+    - pkg-config
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - maturin                             # [build_platform != target_platform]
+    - {{ compiler('c') }}                 # [linux]
+    - {{ compiler('m2w64_c') }}           # [win]
+    - {{ compiler('cxx') }}               # [linux]
+    - {{ compiler('rust') }}
```

Need to add cargo-bundle-licenses too, see https://conda-forge.org/docs/maintainer/adding_pkgs.html#licenses-of-included-dependencies
```suggestion
    - {{ compiler('rust') }}
    - cargo-bundle-licenses
```